README.md

```diff
@@ -129,10 +129,8 @@ A list of weights, each weights definition contains the following fields:
     - `source` link to the weights file. Preferably an url to the weights file.
     - `sha256` SHA256 checksum of the model weight file specified by `source` (see `models` section above for how to generate SHA256 checksum)
     - `timestamp` timestamp according to [ISO 8601](#https://en.wikipedia.org/wiki/ISO_8601)
-    - `test_inputs` relative file path to test inputs. `language` and the file extension define its memory representation.
-    The test inputs are always stored as a list of tensors as described in `inputs`.
-    - `test_outputs` relative file path to test outputs. `language` and the file extension define its memory representation.
-    The test outputs are always stored as a list of tensors as described in `outputs`.
+    - `test_inputs` list of relative paths to test inputs as described in `inputs`. Supported file formats/extensions: `.npy` (depracated), todo: aggree on more formats
```

Comment:
See https://github.com/bioimage-io/configuration/issues/35
I think we can just go for the npy format.

Reply:
I don't think so. Whats wrong with TIFF files?
Or is it super simple to deal with .npy in Java and Fiji? (This is a rhetorical question... ;)

Reply:
Tiff files are not universal: 1) tiff is more like a container with many variations that can yield different results with different readers 2) we cannot use tiff to store test_outputs for, e.g. classification models that output labels. with `npy` we can handle arbitrary dimensions.

I would say npy formats are rather simple to handle, there are already a couple of libraries in java, see https://github.com/bioimage-io/configuration/issues/35.